esx_service/vmdk_ops.py

```diff
@@ -1183,6 +1188,7 @@ def setStatusAttached(vmdk_path, vm, vm_dev_info=None):
     if not vol_meta:
         vol_meta = {}
     vol_meta[kv.STATUS] = kv.ATTACHED
+    # instanceUuid i.e. vc uuid of a VM is stored in volume kv
```

Comment:
Unrelated to this change but need to handle a case where instanceUuid doesnt exist and we fail back to biosUUID